incubator/mariadb-galera/README.md

```diff
@@ -0,0 +1,115 @@
+# MariaDB Galera Cluster
```

Comment:
Please add a NOTES.txt telling people how to connect. A good example can be found in the mariadb chart:
https://github.com/kubernetes/charts/blob/master/stable/mariadb/templates/NOTES.txt